guides/common/modules/proc_configuring-smart-proxy-server-with-custom-ssl-certificates-to-generate-and-sign-puppet-certificates.adoc

```diff
@@ -56,17 +56,31 @@ Retain a copy of the example `{foreman-installer}` command from the output for i
 --puppet-server true \
 --puppet-server-ca "true"
 ----
-. On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxies} that you configure for load balancing, except this first system where you configure Puppet certificates signing:
+. On {SmartProxyServer} that is the Puppetserver Certificate Authority, stop the Puppet server:
 +
-[options="nowrap", subs="+quotes,attributes"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl stop puppetserver
+----
+. Generate Puppet certificates for all other {SmartProxyServers} that you configure for load balancing, except the first system where you configure Puppet certificates signing:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# puppet cert generate _{smartproxy-example-com}_ \
---dns_alt_names=_{loadbalancer-example-com}_
+# puppetserver ca generate \
+--ca-client \
+--certname _{smartproxy-example-com}_ \
+--subject-alt-names _{loadbalancer-example-com}_
 ----
 +
-This command creates the following files on the Puppet certificate signing {SmartProxyServer} instance:
+This command creates the following files on the system where you configure {SmartProxyServer} to sign Puppet certificates:
```

Comment:
```suggestion
This command creates the following files:
```
I don't think you need to specify where the files are created because the content makes it clear.